frontend/src/components/request/request.js

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import Button from '../../components/common/Button/Button';
+import Checkbox from './Checkbox/Checkbox';
+import PropTypes from 'prop-types';
+const RequestBlock = styled.div``;
+
+Request.propTypes = {
+  user: PropTypes.object,
+  fromList: PropTypes.array,
+  toList: PropTypes.array,
+  minimumPassenger: PropTypes.array,
+  onClickRequest: PropTypes.func,
+};
+
+function Request({ user, fromList, toList, minimumPassenger, onClickRequest }) {
+  const [from, setFrom] = useState(null);
+  const [to, setTo] = useState(null);
+  const [minPassenger, setMinPassenger] = useState(null);
+
+  const onCheckboxClickHandler = (name, value) => () => {
+    if (name === 'from') {
+      from === value ? setFrom(null) : setFrom(value);
+    } else if (name === 'to') {
+      to === value ? setTo(null) : setTo(value);
+    } else if (name === 'minimumPassenger') {
+      minPassenger === value ? setMinPassenger(null) : setMinPassenger(value);
+    }
+  };
+
+  const onButtonClickHandler = () => {
+    if (
+      user !== null &&
+      from !== null &&
+      to !== null &&
+      minPassenger !== null
+    ) {
+      onClickRequest({
+        userId: user.id,
+        from: from,
+        to: to,
+        minimumPassenger: minPassenger,
+      });
+    } else {
+      window.alert('Please Check!');
+    }
```

Comment:
if you are checking `user === null || from === null || to === null` case first we can remove `else` statement.

So fixing like as following:

```
if (user === null || ... ) {
  window.alert(...)
  return
}
<do something...>
```

would be more clean!